pages/blog/index.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
```

Comment:
Please move this code into a new file called `Blog.tsx` in this same directory and inside the `index.ts` add the code `export * from './Blog'`